### ISSUE TITLE
Feature- Add image upload to DB for events

### ISSUE THREAD

**Issue by DaveToth77** [brentocracy/kindly#27]
- Add image upload capabilities for user ( upload button?)
- create DB structure to host/dispatch image

**Comment by brentocracy** [comment]
This is going to be a later feature, but we should have a default image uploaded so there is no alt text messing with the UI.
